test/unit/ngsiv2/amqpBinding-test.js

```diff
@@ -65,13 +65,11 @@ describe('AMQP Transport binding: measures', function() {
 
```

Comment:
It would be great to to a grep in test/unit/ngsv2/ for `/v1` to ensure that everything has been changed, as was done in the library PR. Maybe you have done it already :)

Reply:
Done:

```
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-ul$ grep -Rin v1 test/unit/ngsiv2/
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-ul$
```

Reply:
NTC